app/components/ArticlePreview.tsx

```diff
@@ -52,16 +29,8 @@ const ArticlePreview = (props: ArticlePreviewProps) => {
       </div>
       <h3 className="text-left text-2xl line-clamp-2">{title}</h3>
       <p className="my-5 line-clamp-3">{description}</p>
-      <div className="inset-x-6 flex flex-col items-center sm:absolute sm:bottom-2 md:flex-row md:justify-between">
-        <span className="hidden sm:flex md:order-1">
-          <Link to={linkButtonText().route}>{linkButtonText().text}</Link>
-        </span>
-        <a
-          className="order-1 w-full md:order-2 md:w-max"
-          href={url}
-          target="_blank"
-          rel="noreferrer"
-        >
+      <div className="inset-x-6 items-center sm:absolute sm:bottom-2">
```

Comment:
Foreslår at du ikke bruker absolute her, men heller margin og padding (eks: sm:mb-2). Så blir det litt mer konsistent på tvers av applikasjonen 😅 Jeg synes også vi kan ha litt mer margin under knappen for det ser litt rart ut når den er så nærme bunnet av kortet

Reply:
Skjønner at absolute brukes for å få det på bunnen her, men tror vi kan finne en annen tilnærming enn å bruke det her. Men det kan taes i issue om å refaktorere cardWithContent 👍 Så la det stå slik det er foreløpig

Reply:
Forstår tankegangen din, og enig i at vi kan la det være et eget issue :)